### PR TITLE
1.4.1 release notes

### PR DESCRIPTION
**Pull request by reiddraper**

### CODE REVIEW

**Comment by kellymclaughlin** [comment]
:+1:

**Comment by kuenishi** [comment]
May I put Japanese translation to this branch like I did in 1.4.0?

**Comment by kellymclaughlin** [comment]
> May I put Japanese translation to this branch like I did in 1.4.0

:+1:

**Comment by kuenishi** [comment]
Thanks! I'll get translation review and merge this into release/1.4 branch.

File: RELEASE-NOTES.ja.org [hunk 1]
```diff
@@ -1,3 +1,13 @@
+* Riak CS 1.4.1 リリースノート
+** 修正された問題
+- Fix list objects crash when more than the first 1001 keys are in
+  the pending delete state
+- 最初の1001個のキーがpending delete状態だったときにlist objectsがクラッシュ
```
Comment (shino) [inline RELEASE-NOTES.ja.org]: s/1001/1002/

**Comment by shino** [comment]
:+1: after fix for the comment above

**Pull request merged**